pyproject.toml

```diff
@@ -40,6 +40,7 @@ dynamic = [
   "version",
 ]
 dependencies = [
+  "packaging",
```

Comment:
Let's freeze this to a particular version. Looks like `packaging~=24.0` may work?

Reply:
I'm not too sure how to best handle this requirement. But from looking at [matplotlib](https://github.com/matplotlib/matplotlib/blob/main/pyproject.toml#L39C1) and [tox](https://github.com/tox-dev/tox/blob/main/pyproject.toml#L55), it looks like it's fine not restricting it to some major version. So this is fine 👍